### DISCUSSION TITLE
if else loop in orquesta workflow

### DISCUSSION THREAD

**Discussion by komalphadatare** [StackStorm/st2#5430] [Workflows]
How i can implement the if else loop in stackstorm. 
eg : if the output of an action is  'xyz'
       Then: go to Transition1
       else if the output of an action is 'abc'
       Then: go to Transition2   
       else
       Then: go to Transition2   

We tried with solutions available below link, but no luck.
1. https://github.com/StackStorm/orquesta/issues/116
2. https://forum.stackstorm.com/t/if-statement-in-orquesta/1158/5
3. https://github.com/StackStorm/st2/wiki/Orquesta-Implementation-Patterns

we are Using st2 3.1.0, on Python 2.7.5 Version

**Comment by minsis** [comment]
Have you looked here for examples? https://docs.stackstorm.com/orquesta/languages/orquesta.html#task-transition-model
